src/common.h

```diff
@@ -96,6 +96,10 @@ typedef unsigned long long uint64;
 typedef signed long long int64;
 #endif
 
+#ifdef _MSC_VER
+#pragma warning(disable:4100) /* unreferenced formal parameter */
+#endif
+
```

Comment:
I don't know if it matters that much, but is there a way to provide this from the command line the same way we do `-Wno-unused-parameter` for GCC?